src/main/java/com/nha/abdm/wrapper/hrp/discoveryLinking/requests/OnConfirmRequest.java

```diff
@@ -0,0 +1,71 @@
+package com.nha.abdm.wrapper.hrp.discoveryLinking.requests;
+
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+//import com.nha.abdm.wrapper.hrp.CareContextService;
+import com.nha.abdm.wrapper.hrp.serviceImpl.LogsTableService;
+import com.nha.abdm.wrapper.hrp.serviceImpl.PatientTableService;
+import com.nha.abdm.wrapper.hrp.common.Utils;
+import com.nha.abdm.wrapper.hrp.discoveryLinking.responses.ConfirmResponse;
+import com.nha.abdm.wrapper.hrp.hipInitiatedLinking.responses.LinkRecordsResponse;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.stereotype.Component;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.UUID;
+
+@Component
+public class OnConfirmRequest {
```

Comment:
These class should just be a helper class to create a request object from the arguments passed to it. Logics like fetching ABHA address or CareContexts should reside in the Service classes instead.

Reply:
Implemented the Helper classes in such a way that they wont handle any business logic, and it has been done.